README.md

```diff
@@ -103,4 +103,25 @@ yarn gen:types
 ```
 
 If you have faced the error above - check this fix and its description:
-https://github.com/cowprotocol/bff/pull/101
\ No newline at end of file
+https://github.com/cowprotocol/bff/pull/101
+
+# Development notes
+
+## Notifications
+To run locally the notifications, you will need to run:
+- RabbitMQ locally: `docker-compose up -d queue`
+- DB locally: `docker-compose up -d db`
+- Telegram consumer: `yarn telegram`
+- Notification producer: `yarn producer`
```

Comment:
Nitpick/suggestion: add a single docker-compose file for running all services locally from a single command

Reply:
I think we have, the instructions try to be a bit more explicit on how to launch individual ones, but probably we should also make it a oneliner as you say